tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -37,6 +37,22 @@
              stored in the .snirf format.
 
 
+.. _import-boxy:
+
+BOXY (.txt)
+================================
+
+BOXY recordings can be read in using :func:`mne.io.read_raw_boxy`.
+The BOXY software and ISS Imagent I and II devices store data in a single .txt
+file containing DC (overall background light intensity),
```

Comment:
Following our previous discussion about this DC component being the same as the continous wave amplitude measurement then this explanation does not make sense to me. Background light implies to me that its something to do with the ambient light in the room, and in a CW system you wouldnt say that the measurement is of the background intensity.